README.md

```diff
@@ -106,8 +114,9 @@ This will create a virtual environment `venv` in your iBridges folder. Next to t
 
 For Windows simply click on the file `output/ibridgesgui/ibridges_gui.exe`.
 
-Please note, that this feature is still in experimental state.
- 
+## Extending iBridges-GUI (for developers)
+iBridges-GUI suuports Plugins. You can create your own tab and it will be detected and run by the main application.
```

Comment:
```suggestion
iBridges-GUI supports Plugins. You can create a plugin for your own tab and it will be detected and run by the main application when it is installed.
```